packages/@yoda/network/index.js

```diff
@@ -0,0 +1,316 @@
+'use strict'
+
+/**
+ * @module @yoda/network
+ * @description Provides classes to manage network functions on the device.
+ */
+
+var crypto = require('crypto')
+var EventEmitter = require('events').EventEmitter
+var FloraComp = require('@yoda/flora/comp')
+
+/**
+ * @class
+ * @auguments EventEmitter
+ * @param {boolean} subscribe - subscribe network.status or not
+ * @memberof module:@yoda/network
+ */
+class NetworkAgent extends EventEmitter {
+  constructor (subscribe) {
+    super()
+
+    this._callTarget = 'net_manager'
+    this._callCommand = 'network.command'
+    this._callTimeout = 60 * 1000
+
+    this._fid = '@network' + crypto.randomBytes(2).toString('hex')
+    this._flora = new FloraComp(this._fid)
```

Comment:
Use flora agents from`@yoda/flora` directly instead of `@yoda/flora/comp` since it's designed for long running services.